src/examples/radorder.json

```diff
@@ -1438,7 +1438,7 @@
               "linkId": "previousResults.attachment.data",
               "definition": "http://fhir.ch/ig/ch-rad-order/StructureDefinition/ch-rad-order-media#Media.content.data",
               "text": "Daten",
-              "type": "string"
```

Comment:
does this file get updated regularly?

Reply:
We will see. There will be some updates but they are always ratified by ahdis.